.github/workflows/playwright.yml

```diff
@@ -0,0 +1,42 @@
+name: Playwright Tests
+on:
+  push:
+    branches:
+      - master
+      - 'release/**'
+  pull_request:
+    branches:
+      - master
+      - 'release/**'
+env:
+  # Warnings breaks build on CI
+  # See: https://github.com/orgs/community/discussions/25228#discussioncomment-3246960
+  CI: false
+  MODE: standalone
+  KETCHER_URL: http://frontend:4002
+jobs:
+  playwright_tests:
+    timeout-minutes: 60
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.12.0
+      - name: Install dependencies
+        run: npm ci
+      - name: Build all packages
+        run: npm run build
+      - name: Build frontend and autotests for docker
+        run: npm run docker:build
+      - name: Run playwright tests in docker
+        run: cd ketcher-autotests && npm run docker:test
```

Comment:
This should probably be npm run docker:test:**ci** ?

Reply:
fixed